rollup.config.executor.js

```diff
@@ -0,0 +1,79 @@
+//import commonjs from '@rollup/plugin-commonjs';
+//import hash from 'rollup-plugin-hash';
+import outputManifest from 'rollup-plugin-output-manifest'
+import typescript from '@rollup/plugin-typescript';
+
+
+var outDir = 'build/resources/main/assets/react4xp';
+
+export default {
+	external: [
+		//'prop-types',
+		'react',
+		'react-dom'
+	],
+	input: 'src/main/resources/assets/react4xp/executor.ts',
+	output: {
+		dir: outDir,
+
+		entryFileNames: '[name].[hash].js',
+
+		//format: 'cjs',
+		//format: 'es',
+		//format: 'esm',
+		//format: 'iife',
+		format: 'umd',
+
+		globals: {
+			//'prop-types': 'PropTypes',
+			'react': 'React',
+			'react-dom': 'ReactDOM'
+		},
+
+		name: 'React4xp.EXECUTOR', // bundle name (required by at least umd) // But I'm not seeing it in the resulting file? But if it works, who cares...
```

Comment:
It works although it doesn't appear - but this line is still necessary?

Reply:
Dunno, don't care.